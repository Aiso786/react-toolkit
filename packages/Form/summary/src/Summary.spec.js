import React from 'react';
import renderer from 'react-test-renderer';
import { Summary } from './index';

const messages = ['Field Author is required', 'Field PlaceName is required'];

describe('<Summary>', () => {
  it('renders Summary correctly', () => {
    const tree = renderer
      .create(
        <Summary messages={messages} title="Le formulaire n'est pas valide" />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
