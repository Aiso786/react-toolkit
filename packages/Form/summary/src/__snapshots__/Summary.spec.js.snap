// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Summary> renders Summary correctly 1`] = `null`;
